### ISSUE TITLE
Work with citations

### ISSUE THREAD

**Issue by leenapthine** [etcluvic/hsscommons#252]
(~~)1. Looking at the individual publications there is a 'citations' tab, and a 'cite this work' component in the 'About' section. We want to remove the citations tab, and any other references to citations in the publication section. Likely this can be done in the backend. (~~)

ex: https://test.hsscommons.ca/en/publications/3171/1

(~~)2. We want to preserve the 'Cite this work' section, and add a `copy` button that will allow users to copy the citations directly without needing to highlight and copy manually.(~~)

(~~)3. Look into an export option for users to create both .JSON and .CSV exports of the 'cite this work' section.(~~)

4. Under the 'Master Types' menu in the admin (components->publications->master types->files) there are block configurations. Add additional blocks for info fields such as 'year', 'journal title', etc. Check if this information gets added to the html/metadata in 'view page source'. If it doesn't, find out how to to get it added. Ex. 'Year of publication', 'language'.

**Comment by ghjensen** [comment]
Re: the third task above, one possible future wish-list item (which we can discuss further if we get to that point) would be to provide export data in **RO-Crate** format. See https://www.researchobject.org/ro-crate/about_ro_crate. This is a format used by many of our Australian partner organizations and could facilitate some interesting interoperability possibilities.

**Comment by leenapthine** [comment]
Plugging away at these.

1. This is an easy fix that can be done on the backend. 
2. I have a PR in review for this issue. The button works and copies citations, though may still want some styles. 
3. This issue is also in PR awaiting review. 
4. I have looked into this. These meta attributes can't be added by the 'master types' blocks. I can add meta tags to the html easily enough, but we would need the tags to point to specific elements in the _publications table in the database. I think this will require adding attributes to the database tables and I think we can wait on this until the new year.

**Comment by leenapthine** [comment]
1., 2., and 3. have been merged into production. I will this open until 4. has been completed.

**Comment by leenapthine** [comment]
This has been worked on further to properly format the RO Crate exports.

**Comment by ghjensen** [comment]
Thanks, Lee! This looks great.
